poly-synth/src/poly-synth.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+class PolySynth {
+  /**
+   * @class  PolySynth
+   * @constructor
+   * PolySynth manages the life cycle of voices.
+   * @param {AudioContext} context the audio context
+   */
+  constructor(context) {
+    if (!(context instanceof AudioContext))
+      throw context + ' is not a valid audio context.';
+
+    this.context_ = context;
+    this.sustainedVoices_ = [];
+    this.releasedVoices_ = [];
+
+    // The default maximum cutoff is set near the upper edge of human hearing
+    // and the minimum cutoff is near the lower edge of human hearing.
+    this.lowPassMaxCutoff = 12000;
+    this.lowPassMinCutoff = 60;
+
+    // By default, the filter should not affect audible frequencies.
+    this.lowPassCutoff_ = this.lowPassMaxCutoff;
```

Comment:
12 kHz cuts out a lot of audible frequencies.  (Well, not me, but teenagers can hear 18 kHz or more.)  So either the comment is wrong or the max cutoff is wrong.